### PR TITLE
WNMGDS 814/e2e tool update

### PR DESCRIPTION
**Pull request by scheul93**
## Summary
Updating the npm package we use for axe accessibility testing

## How to test
- Pull down branch
- run `yarn install`
- run `yarn test`

_Note: it is expected that the e2e tests will fail. Specifically, one of the spinner tests. This is good! This error was also caught by DK in [this Jira](https://jira.cms.gov/browse/WNMGDS-843) The fix for this test will come in a separate PR and that is a dependency for merging this PR_

### CODE REVIEW

File: packages/design-system-scripts/helpers/e2e/assertNoAxeViolations.js [hunk 1]
```diff
@@ -41,23 +41,25 @@ function printViolations(violations) {
   violations.forEach(printViolation);
 }
 
-module.exports = async function assertNoAxeViolations(url, disabledRules = []) {
+module.exports = function assertNoAxeViolations(url, disabledRules = []) {
   if (url) {
-    await driver.get(url);
+    return driver.get(url)
```
Comment (scheul93) [inline packages/design-system-scripts/helpers/e2e/assertNoAxeViolations.js:46]: I re-structured this because I think this is partially the cause for some of the timeouts. I couldn't figure out where it was happening in the async/await syntax, but using promises seems to work.

**Review by line47 — APPROVED** [review]
This works well for me! (I did need to update ChromeDriver to 93)

File: packages/design-system-scripts/jest/unit.config.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ module.exports = (rootDir, core) => ({
   snapshotSerializers: ['enzyme-to-json/serializer'],
   testPathIgnorePatterns: ['dist/', 'node_modules/', '.+\\.e2e\\.test\\.js$'],
   transformIgnorePatterns: ['node_modules(?!/@cmsgov)'],
+  testTimeout: 30000,
```
Comment (pwolfert) [inline packages/design-system-scripts/jest/unit.config.js:12]: Is this related? Are unit tests timing out?

**Review by pwolfert — COMMENTED** [review]
E2E tests are working for me now 👍. One question below

File: packages/design-system-scripts/jest/unit.config.js (hunk 2 above)
Comment (scheul93) [reply packages/design-system-scripts/jest/unit.config.js:12]: Ah! This is a leftover. Updating...

**Review by pwolfert — APPROVED** [review]
LGTM 👍
